api/envoy/extensions/filters/http/jwt_authn/v3/config.proto

```diff
@@ -117,6 +117,13 @@ message JwtProvider {
     //      inline_string: ACADADADADA
     //
     config.core.v3.DataSource local_jwks = 4;
+
+
+    // The two below fields define the amount of slack in seconds that will be used
+    // when determining if a JWT is valid or has expired. Validity is determined by
+    // the formula: VALID_FROM("iat") - nbf_slack_slack <= NOW <= VALID_TO("exp") + exp_slack.    
```

Comment:
Where does this formula come from? RFC?

Reply:
This formula is about token validity and it is not mentioned in https://tools.ietf.org/html/rfc6749. But let me make my case:
///////////////////////////////////////////////////
Some load balancer performinc OIDC authentication receives a request and determines that the existing access token is valid (for another 1 second). It forwards the request to Istio Ingressgateway and subsequently to some pod with an envoy sidecar. Meanwhile, 
1 second has passed and when envoy checks the token it finds that it has expired.
///////////////////////////////////////////////////

Reply:
About int32/uint32: I think we should use int32 since what we are trying to do here is provide a configuration option to users. If someone f.e. wants to use nbf_slack = -60 in order to ensure that tokens are valid, starting one minute into the future after their time of issue - why not?